trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super[0..9]
+  end
+end
```

Comment:
- [ ] This was supposed to be pointed out by the previous reviewers, you are meant to remove this Microverse logo and the text as they are placeholders that came with the template.You can add your logo if you have any or leave out the logo entirely. :small_airplane: :wave: 

- - Screenshot:
![Screenshot from 2023-08-29 10-44-25](https://github.com/rabayarabu/OOP-school-library/assets/107632780/be282876-1478-4b61-9733-987c595b4067)